pkg/resource/parameter_group/hooks.go

```diff
@@ -158,6 +159,141 @@ func (rm *resourceManager) resetAllParameters(
 	return &resource{ko}, nil
 }
 
+// customUpdate overrides sdkUpdate by custom logic
+func (rm *resourceManager) customUpdate(
+	ctx context.Context,
+	desired *resource,
+	latest *resource,
+	delta *ackcompare.Delta,
+) (updated *resource, err error) {
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("rm.sdkUpdate")
```

Comment:
Please update this to `customUpdate` instead of `rm.sdkUpdate`

Reply:
Modified.